### ISSUE TITLE
[RESEARCH] Begin research planning for Mobile search expectations (assistive tech) (v1) 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#18685]
## User Story 

- As a Veteran **_using assistive tech and a mobile device_** to search on Facility Locator, I want the experience to be consistent with accessibility best practices. 
- As the Facilities team, we want the Facility Locator to be easy to use with assistive technology so the search experience is an equitable experience for all Veterans.

## Background 

- In a mobile month study, veterans were not interested in using the map view of facility locator #14484
- Recent changes have been made to resolve accessibility defects and implement two new usability features ("find my location" and "clear field"). We will conduct this research in the staging or prod environment since assistive tech typically cannot be used with wireframes and prototypes
- The Facility Locator search experience has not been approached from a mobile-first perspective
- A separate mobile session will be conducted with a general Veteran population #5761 

## Goals of research

- Establish foundational knowledge for the search experience for Veterans using assistive tech (with a mobile perspective)
- Validate recent keyboard navigation and voiceover changes
- Identify pain points for Veterans using assistive tech to search for facilities on a mobile device
- Understand how Veterans expectations for search results differ for mobile vs desktop users 
- Understand how industry standards impact Veteran expectations for the Facility Locator search experience

## Questions

**By keyboard/voice/with low vision/hearing and/or with cognitive impairment....**
**When searching on mobile device**
- Veterans using assistive tech have access to all functionality, can distinguish the interactive elements

**Search box**
- Is the search box easy to navigate? (Forward and back, no keyboard traps)
- What is the appropriate search experience for the typeahead search field (Facility type = Community care providers) 
- Do participants understand how to enable their location?
- Do participants understand how to clear inputs?
- Do error messages provide adequate support? (in particular, when searches return no results due to the radius limit)
- What are the pain points?

**Map**
- Can the Veteran easily interact with the map?
- Are the context changes clearly communicated?
- Do the pins provide adequate information about the location?
- Is the map able to be navigated in a logical order?

**Search results**
- How many results do Veterans expect?
- For searches with a high number of results (ie: community pharmacy), is the number of tab stops/information presented appropriately?
- Are results presented in logical reading order?
- Can the result list be navigated in a logical order?
- Can the Veteran navigate easily between the results and map?
- Do Veterans using assistive tech on a mobile device prefer to view results on the map or in a list?
- Does the information presented for a given facility match expectations?
  -  Does the Veteran understand how to find more detailed information about the facilities returned?
- What are the pain points?
- What online maps/searches do these participants find easiest to use/why?

## Tasks
- [ ] Work with Accessibility team to shape/iterate recruitment strategy, research design, accessibility questions, etc.  
- [ ] Create research plan v1

## Acceptance Criteria
[ ] Research plan v1

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mmiddaugh** [comment]
- Research plan is available [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/research/user-research/screenreader-usability-study/research-plan.md)
- Conversation guide will be created during Sprint 46 #23493
